python/podio_class_generator.py

```diff
@@ -156,6 +178,17 @@ def process_schema_evolution(self):
           print(warning)
         sys.exit(-1)
 
+      # now go through all the io_handlers and see what we have to do
+      if 'ROOT' in self.io_handlers:
+        for item in root_filter(comparator.schema_changes):
+          schema_evolutions = self.root_schema_dict.get(item.klassname)
```

Comment:
This should replace the entire loop body
```python
# add whatever is relevant to our ROOT schema evolution
self.root_schema_dict.setdefault(item.klassname, []).append(item)
```
Or use a `defaultdict`

Reply:
good observation. simplified now.